cnf-tests/testbinaries/oslat-runner/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"flag"
+	"os/exec"
+	"strings"
+	"time"
+
+	"k8s.io/klog"
+	"k8s.io/kubernetes/pkg/kubelet/cm/cpuset"
```

Comment:
do we want to use a simpler package (less complex dep tree) like https://github.com/fromanirh/cpuset/ ?

Reply:
indeed my own `cpuset` is intentionally a bare bones version, exactly to keep the baggage minimal. And if we already pull the kube deps no reason not to use the full-featured cpuset package, no problems.